backend/src/validators/json_validator.py

```
@@ -1,38 +1,61 @@
+- Fix array bracket issues
+- Ensure valid JSON structure
+
+Return ONLY the cleaned JSON, no other text."""
+
+        try:
+            import asyncio
+            try:
+                asyncio.get_running_loop()
+                json_str = json_str.replace("'", '"')
+                logger.debug(f"Basic cleaned JSON: {json_str}")
+                return json_str
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)